src/Login/Login.test.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
-import Header from '../Header/Header.js'
 import Login from '../Login/Login.js'
-import { screen, fireEvent, render } from '@testing-library/react';
+import { screen, render } from '@testing-library/react';
```

Comment:
The test set up you have that would use the fireEvent look solid - definitely a great opportunity to test what happens on a button click.